### ISSUE TITLE
Clarify workflow at first launch

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-android#3943]
It seems to me, from the analysis of PlayStore comment, that a significant amount of our users are a bit clueless at first start. They don't understand or don't achieve to run successfuly the following steps:
* Go to online library ONLINE
* Download catalogue of books ONLINE
* Choose and download a book ONLINE
* Enjoy the book(s) OFFLINE

I open this issue to:
* Gather feedbacks on this opinion
* If confirmed, gather ideas of solutions

**Comment by Popolechien** [comment]
This is a recurring issue I think with Spanish and Arabic speakers, seeing as we get lots of mostly empty messages.

I think there should be an onboarding wizard to explain what they're supposed to expect.

**Comment by MohitMaliFtechiz** [comment]
I agree with the @Popolechien option, a tour for users at the first launch would help them to understand these functionalities. As we are educating users on the file transfer functionality that we introduced in https://github.com/kiwix/kiwix-android/pull/3497.

**Comment by gouri-panda** [comment]
Hello @kelson42 ! Yes, I agree with @Popolechien @MohitMaliFtechiz! To ensure the app is user friendly and easy to understand, we should focus on several key areas:

**Intuitive UI Design:**   We should use consistent design patterns and prioritise important information to create a clear visual hierarchy. Navigation will be intuitive and logical, with familiar icons and gestures. As for my knowledge, there is a lot of room for navigation improvement. For example, first time users don't know where to go or what to do with it. We should also think about the `Download Books` and  `Open Library`  buttons from the corresponding screens. From my standpoint it clearly misleads. For e.g if the Reader section is empty and my library section is empty why is it showing `Open Library` button, it should show us the `Download Books` button. Also, i think there should be a search button in the library section to search the zim files from the server(like in the Download section) for better UX.

**Onboarding Process:**  We will provide simple tutorials and use progressive disclosure to introduce features gradually. Currently, we're just doing in one screen at onboarding. In this new approach, we can introduce the app featues/ use cases gradually when they're main screen. Additionally, we can add the tutorials in the help fragment.


These are small improvements that we can implement quickly.

**Comment by rgaudin** [comment]
Do we still have a minimal embedded ZIM when downloading the app? Was Ray Charles when we introduced it if I recall correctly.

**Comment by MohitMaliFtechiz** [comment]
@rgaudin No, now we do not have the embedded zim file for Kiwix app.

**Comment by soloturn** [comment]
i started kiwix first time, and it says "download books" then opens the screen below. i d love to select the languages i speak, or popular languages, to find popular content, rated with stars, number of downloads, a paragraph explaining the contents. sort by size, with w/o media, categories. something like kiwix desktop download manager feeling, a little.

<img width="920" height="2048" alt="Image" src="https://github.com/user-attachments/assets/2e3726e0-bf56-44a8-a594-74109efa9716" />
